service/Cargo.toml

```diff
@@ -27,8 +27,7 @@ futures = "0.3"
 
 # ipfs
 ipfs-api = "0.11.0"
-multihash = "0.8"
-cid = "<0.3.1"
```

Comment:
@clangenb According to git you added this quite specific version number for cid. Do you remember why?

Reply:
Oh this was ages ago, in my young times. Now, I would scold everyone who does not documents such a thing :D.

I think this was either due to:
* That it was incompatible with sgx at that time (different std-lib interfaces)
* There was some format change

I kind of remember that it was the latter, although I am not even sure if that makes sense. Anyhow, this is so old, I suggest upgrading and then fix the IPFS if this is needed.